adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala

```diff
@@ -327,8 +327,23 @@ case class FeatureRDD(rdd: RDD[Feature],
    * @return Since a feature maps directly to a single genomic region, this
    *   method will always return a Seq of exactly one ReferenceRegion.
    */
-  protected def getReferenceRegions(elem: Feature): Seq[ReferenceRegion] = {
-    Seq(ReferenceRegion.unstranded(elem))
+  protected def getReferenceRegions(elem: Feature,
+                                    stranded: Boolean = false): Seq[ReferenceRegion] = {
+    stranded match {
+      case true => {
+        Seq({
+          try {
+            ReferenceRegion.stranded(elem)
+          } catch {
+            case e: IllegalArgumentException => ReferenceRegion.unstranded(elem)
```

Comment:
`ReferenceRegion.stranded(feature: Feature)` has a require that the `Feature` have stranded data.